src/services/navigator/helpers.test.ts

```diff
@@ -2,7 +2,7 @@ import * as DomErrors from 'hyperview/src/services/dom/errors';
 import * as Errors from './errors';
 import * as Namespaces from '../namespaces';
 import * as Types from './types';
-import * as TypesLegacy from '../../types-legacy';
+import * as TypesLegacy from '../../types';
```

Comment:
Can we correct these paths now that the tsconfig is updated?

Reply:
let's address this separately after the migration is merged on master